monai/networks/nets/spade_network.py

```
@@ -156,12 +156,12 @@ def __init__(
         self.z_dim = z_dim
         self.channels = channels
         if len(input_shape) != spatial_dims:
-            raise ValueError("Length of parameter input shape must match spatial_dims; got %s" % (input_shape))
+            raise ValueError(f"Length of parameter input shape must match spatial_dims; got {input_shape}")
         for s_ind, s_ in enumerate(input_shape):
             if s_ / (2 ** len(channels)) != s_ // (2 ** len(channels)):
                 raise ValueError(
                     "Each dimension of your input must be divisible by 2 ** (autoencoder depth)."
-                    "The shape in position %d, %d is not divisible by %d. " % (s_ind, s_, len(channels))
+                    f"The shape in position {s_ind}, {s_} is not divisible by {len(channels)}. "
                 )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)